modules/aws/vpc/vpc.tf

```diff
@@ -51,7 +51,7 @@ EOF
       "Resource": "arn:${var.arn_region}:s3:::etcd-backups.giantswarm.io/*"
     },
     {
-      "Sid": "CAPA",
+      "Sid": "CAPA-Rule",
```

Comment:
Why are we changing this ? I prefer to avoid unnecessary changes.

Reply:
I'll revert it on monday. It just makes non sense that this rule does not end with -Rule when all the others do

Reply:
Done